### PR TITLE
update 101: add faceting and pagination

### LINKED ISSUE #1717: Update Meilisearch 101
- Update [Getting ready for production](https://docs.meilisearch.com/learn/getting_started/getting_ready_for_production.html) to add multitenancy and tenant tokens
- Update [Fine-tuning search results](https://docs.meilisearch.com/learn/getting_started/customizing_relevancy.html) to add faceting and pagination

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1717

### CODE REVIEW

File: learn/getting_started/customizing_relevancy.md [hunk 1]
```diff
@@ -149,4 +149,28 @@ The above code sample sets the minimum word size for one typo to `4` characters.
...
+
+## Pagination
+
+By default, Meilisearch returns 1000 results per search. This limit protects your database from malicious scraping.
+
+The code sample below sets this limit to 500:
+
+<CodeSamples id= "getting_started_pagination" />
+
+Now, you won't be able to access search results beyond 500.
```
Comment (dichotommy) [inline learn/getting_started/customizing_relevancy.md:172]:
```suggestion
Now, users won't be able to access search results beyond 500.
```

File: learn/getting_started/customizing_relevancy.md [hunk 2]
```diff
@@ -149,4 +149,28 @@ The above code sample sets the minimum word size for one typo to `4` characters.
 
 You can read more about typo tolerance in our [dedicated guide](/learn/configuration/typo_tolerance.md).
 
+## Faceting
+
+Meilisearch allows you to create faceted search interfaces, refining search results based on broad categories called facets. Like filters, you need to add the attributes you want to use as facets to `filterableAttributes`.
+
```
Comment (dichotommy) [inline learn/getting_started/customizing_relevancy.md:155]:
```suggestion

When creating a faceted search interface, it is helpful to display how results are distributed between the different categories. For example, a user might want to know how many of their results have a `"genre"` of `"comedy"` and how many have a `"genre"` of `"horror"`. Meilisearch supports this with [the `facets` search parameter](/reference/api/search.md#facets).

```

File: learn/getting_started/customizing_relevancy.md [hunk 3]
```diff
@@ -149,4 +149,28 @@ The above code sample sets the minimum word size for one typo to `4` characters.
...
+You can configure the maximum number of facet values returned for each facet.
```
Comment (dichotommy) [inline learn/getting_started/customizing_relevancy.md:156]:
```suggestion
Using the faceting settings, you can configure the maximum number of facet values returned for each facet.
```

File: learn/getting_started/customizing_relevancy.md [hunk 4]
```diff
@@ -149,4 +149,28 @@ The above code sample sets the minimum word size for one typo to `4` characters.
...
+
+Suppose a query's search results for the `genres` facet contains: `Action`, `Adventure`,`Science Fiction`, and `Comedy`. The following code sample will only return `Action` and `Adventure` in the response body's `facetDistribution` object.
```
Comment (dichotommy) [inline learn/getting_started/customizing_relevancy.md:158]:
```suggestion
Suppose your results contain the following values for the `genres` facet: `Action`, `Adventure`, `Science Fiction`, and `Comedy`. After running the below code sample, you would only see how results are distributed between the `Action` and `Adventure` genres.
```

**Review by dichotommy — CHANGES_REQUESTED** [review]
Requested a few changes to make the content more clear for beginners.

Observation: the `faceting` and `pagination` settings do not really help users fine-tune their search results or relevancy. Although this guide has thus far been comprehensive in how it presents the different settings, it may not make sense to continue doing that in the future.

File: learn/getting_started/customizing_relevancy.md [hunk 5]
```diff
@@ -149,4 +149,30 @@ The above code sample sets the minimum word size for one typo to `4` characters.
 
 You can read more about typo tolerance in our [dedicated guide](/learn/configuration/typo_tolerance.md).
 
+## Faceting
+
+Meilisearch allows you to create faceted search interfaces, refining search results based on broad categories called facets. Like filters, you need to add the attributes you want to use as facets to `filterableAttributes`.
+
+When creating a faceted search interface, it is helpful to display how results are distributed between the different categories. For example, a user might want to know how many of their results have a `"genre"` of `"comedy"` and how many have a `"genre"` of `"horror"`. Meilisearch supports this with [the `facets` search parameter](/reference/api/search.md#facets).
```
Comment (maryamsulemani97) [inline learn/getting_started/customizing_relevancy.md:156]:
```suggestion
When creating a faceted search interface, it is helpful to display how results are distributed between the different categories. For example, a user might want to know how many of their results have a `genre` of `comedy` and how many have a `genre` of `horror`. Meilisearch supports this with [the `facets` search parameter](/reference/api/search.md#facets).
```

**Review by dichotommy — APPROVED** [review]
LGTM 🙆🏻

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**